app/controllers/api/v1/developers_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class DevelopersController < Api::BaseController
+      before_action :set_post, only: %i[show]
+
+      def index
+        developers = Developer.includes(
+          :developer_languages,
+          :languages,
+          :developer_programming_languages,
+          :programming_languages
```

Comment:
Do you need to include all of them?

Reply:
I tried , but it suggest include all on log console. This is example: https://gyazo.com/cc2d8da25ef6587a4d23c3084b7447e2